frontend/src/components/Graph/index.tsx

```diff
@@ -374,4 +374,4 @@ Graph.defaultProps = {
 	onDragSelect: undefined,
 	dragSelectColor: undefined,
 };
-export default Graph;
+export default React.memo(Graph);
```

Comment:
can you please revert this change ... already there is memo in develop branch can you please take the reference from there